Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -276,7 +285,14 @@ open class ComposerVC: _ViewController,
 
     override open func setUp() {
         super.setUp()
-
+        
+        if let cooldownDuration = channelController?.channel?.cooldownDuration,
+           cooldownDuration > 0 {
+            content.countdownDuration = cooldownDuration
+        }
+        
+        checkChannelCountdown()
```

Comment:
Same here, ideally we should only handle the countdown logic in `updateContent()`. And in case the countdown is running, it doesn't do anything. 
```suggestion
```

Reply:
Done